src/pages/index.tsx

```diff
@@ -34,7 +34,14 @@ const Home: NextPage = () => {
   const [isNewEmployeeModalOpen, setIsNewEmployeeModalOpen] = useState(false);
   const [isUpdateEmployeeModalOpen, setIsUpdateEmployeeModalOpen] = useState(false);
   const [employees, setEmployees] = useState<Employee[]>([]);
-  const [editingEmployee, setEditingEmployee] = useState<Employee>({} as Employee)
+  const [editingEmployee, setEditingEmployee] = useState({})
```

Comment:
Pass this `editingEmployee` as prop to UpdateEmployeeModal down below at line 77